components/admin/lessons/AdminLessonsSideBar.tsx

```diff
@@ -13,14 +14,17 @@ export const AdminLessonsSideBar: React.FC<SideBarLessonProps> = ({
   setSelectedLesson,
   selectedLesson
 }) => {
-  const lessonListData = lessons || []
-
+  const lessonListData = _.cloneDeep(lessons) || []
+  if (lessonListData.length === 0) {
```

Comment:
Without this check AdminLessonSideBar component won't be able to render empty array of lessons, `lastIndex` will be `-1` and you can get `title` value of undefined resulting in error. This probably won't happen in production (we are not going to delete all lessons), but since tests should cover 100% I needed to add it.